superset/assets/src/explore/exploreUtils.js

```diff
@@ -45,8 +45,10 @@ export function getURIDirectory({
     directory = '/superset/explore_json/';
   }
   // const buildQueryRegistry = getChartBuildQueryRegistry();
-  if (formData.viz_type === 'word_cloud' && !forceExplore) {
-    directory = '/api/v1/query/';
+  if (formData.viz_type === 'word_cloud' ||
+        formData.viz_type === 'table' &&
```

Comment:
Perhaps worthwhile defining an array of `viz_type`s to "whitelist" for the query endpoint and checking that the `viz_type` from `form_data` is in that array (since we'll be adding new ones soon)?